### DISCUSSION TITLE
User Authorization

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#10268] [General - Components / Directives / etc]
Does Quasar have any good solution for 'authentication support'. 'I'm looking to develop an application and really want to use Quasar instead of Vuetify which I used to use. Previously when developing apps in Vueitfy I would use this module > https://auth.nuxtjs.org/ which made the implementation of user authorization super easy. I was wondering if there is a similar solution for an app built in Quasar. It's saves a tremendous amount of setup time.
Thanks again.

**Comment by stefanvanherwijnen** [comment]
Possibly in the future: https://github.com/quasarframework/quasar/discussions/10053 (I´d suggest filling the survey to leave some feedback)
